index.html

```diff
@@ -15,9 +15,11 @@
 <body>
     <main>
         <div class="landingTopContainer">
-            <div class="landingLeft"></div>
+            <div class="stopwatchContainer">
+                <p class="stopwatchText">Stopwatch: <span id="stopwatchSecondsDisplay">0</span> seconds</p>
```

Comment:
i think it will look better (and solve some problems) if instead of:
```
Stopwatch: 6.2 seconds
```

it says:
```
Time: 6.2s
```

Reply:
Made the change.